### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by srezasm** [coinables/coldsigner#2]
Please implement raw(unsigned) transaction hex generation, So it could be signed by the original cold wallet like electrum, samourai, etc.

**Comment by coinables** [comment]
Just to make sure I'm understanding. So you're asking to be able to build an unsigned transaction with coldsigner and output that in hex format so that it could be signed later in another app or wallet?

**Comment by srezasm** [comment]
yes

**Comment by coinables** [comment]
I could do it for signing Legacy addresses, addresses that start with 1. 

For segwit addresses (bech32 or p2sh) you need the scriptpubkey to include them as an input to a transaction.  Currently, for segwit addresses to get around that issues I use the WIF key to re-create the scriptpubkey so that the user does not have to provide it upfront. But if user is not providing the WIF then they would have to provide the scriptpubkey instead.  I guess what I'm saying is it would be possible for segwit addresses as well but I'd have to add another input field for the scriptpubkey.

**Comment by srezasm** [comment]
Yes, the input is it. Could you add an input for scriptPubKey too?
I guess for a better experience it would be good to check the start of the address to decide whether the scriptPubKey is required too

**Comment by coinables** [comment]
There's already a separate page for each address type since each address type add and sign inputs differently. Since there's a separate page for each address type the detection is done by the user when they choose which HTML file to launch.

**Comment by srezasm** [comment]
BTW I just remembered that what I'm looking for is a PSBT transaction

**Comment by coinables** [comment]
PSBTs are a bit more complicated and to be honest I don't have the time to build it with that format, but I'd accept a pull request if someone else did it.
